src/components/Modal/BaseModal/BaseModal.tsx

```diff
@@ -1,12 +1,23 @@
-import { useEffect, useRef } from "react";
+import { useEffect, useRef, ReactNode } from "react";
 import styles from "./base.module.css";
 import closeIcon from "assets/images/ic_close.png";
+import { CloseModal } from "../../../utils/hooks/useModal";
 
-function BaseModeal({ title, children, variant, closeModal }) {
-  const modalRef = useRef(null);
+export interface ModalProps {
```

Comment:
다른 곳에서도 사용되는 타입이다보니 요렇게 사용해주는게 더 이해하기 좋을 것 같네요~
```suggestion
export interface BaseModalProps {
```